app/src/main/res/navigation/health_pass.xml

```diff
@@ -28,8 +26,8 @@
             android:id="@+id/action_healthPassFragment_to_travelPassFragment"
             app:destination="@id/travelPassFragment" />
         <action
-            android:id="@+id/action_healthPassFragment_to_bcscAuthFragment"
-            app:destination="@id/bcsc_auth" />
+            android:id="@+id/action_healthPassFragment_to_bcscAuthInfoFragment"
+            app:destination="@id/navigation2" />
```

Comment:
I would like to discuss this

Reply:
as discussed, using bcsc auth fragments directly.